assets/css/app.scss

```diff
@@ -169,3 +170,7 @@ body {
   flex: 1 1 0;
   min-height: 0;
 }
+
+main.container {
+  @extend .isolation-isolate;
```

Comment:
Question (non-blocking): Why create the utility class rather than just put `isolation: isolate` here?

Reply:
I started with this, but I've been needing to create stacking contexts in different parts of Skate so I figured I'd just create it as a utility class. Not entirely attached to the utility class method.

Reply:
FWIW I think this goes along nicely with the conventions of the Bootstrap utility classes we've been using more and more, so I'm in favor.

Semi-relatedly, I'm not sure if this is necessarily better, but you could just add this class directly to the `<main>` element in `app.html.heex` rather than using an SCSS mixin. That might break out expectations that styling-related stuff can all be found in the front-end code though.